### PR TITLE
Multiple reporters and multiple outputs

### LINKED ISSUE #285: Use reporters from json, support multiple reports
I have a use case where I want to get the nice native json output (which includes the code coverage) and from that json output, use the built-in reporters to put it in various formats.  The reason, I want to use the json file as the source, is because doing lab over and over for each format can potentially give different results (think unreliable downstream dependency) but is also obviously wasteful to re-test again.  It is nice having the 1 data source as the run result dataset, and then printing out whichever reporter you want (it would be nice to include console, html (coverage), junit, etc).  I don't care if i have to run each of these commands separately as they should be fast once the json file is complete.  I am open to really any ideas here.  The goal is to have 1 data set from a run, and use the internal reporters to format it based on that data set.

You could expose the functions so I could write another module or natively support this.  Let me know what you think and what the issues may be with doing this.

### PR DESCRIPTION
**Pull request by hofan41**
Closes #285

### CODE REVIEW

File: lib/reporters/index.js [hunk 1]
```diff
@@ -21,6 +21,10 @@ internals.protos = {
 
 internals.requireReporter = function (reporter) {
 
+    if (Array.isArray(reporter)) {
+        return require('./multiple');
```
Comment (geek) [inline lib/reporters/index.js]: Add a comment to the top of the file that indicates that multiple is required further down.  Just easier to look at the module section to find dependencies

```js
// ('./multiple') loaded below
```

File: lib/reporters/multiple.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+// Load modules
+
+var Path = require('path');
+var Reporters = require('./index');
```
Comment (geek) [inline lib/reporters/multiple.js]: Please move Reporters to be required last in the list
```
Path
Hoek
Reporters
```

We try to do:
```
Node core
3rd Party
Local
```

**Comment by geek** [comment]
@hofan41 Please update the readme with information on how to use this enhancement

**Comment by geek** [comment]
@hofan41 really well done, this is awesome!

**Comment by hofan41** [comment]
Glad I could help! :thumbsup:

**Pull request merged**